packages/webdriverio/src/utils/findStrategy.ts

```diff
@@ -168,15 +168,22 @@ export const findStrategy = function (selector, isW3C, isMobile) {
     }
     case 'name': {
         if (isMobile || !isW3C) {
+            const match = selector.match(/^\[name=("|')([a-zA-z0-9\-_.@=[\] ']+)("|')]$/)
+            if (!match) {
+                throw new Error('InvalidSelectorStrategy') // ToDo: move error to wdio-error package
```

Comment:
Can we have a more explicit error message here? Also I don't think we need to move this error anywhere, at least I would not know where.

Reply:
Sorry for the "todo", blind copy-paste effect.

I think "invalid selector strategy" describes well this, because if `match` is `null` it means that the selector didn't match any strategy, right? What do you think?

Reply:
some more information about the selector that was used and is invalid would be good

Reply:
I've pushed some changes including more info on those errors!